d/desktop-file-utils/stone.yml

```diff
@@ -0,0 +1,24 @@
+#
+# SPDX-FileCopyrightText: © 2020-2024 Serpent OS Developers
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+name        : desktop-file-utils
+version     : 0.27
+release     : 1
+homepage    : https://www.freedesktop.org/wiki/Software/desktop-file-utils/
+upstreams   :
+    - https://www.freedesktop.org/software/desktop-file-utils/releases/desktop-file-utils-0.27.tar.xz : a0817df39ce385b6621880407c56f1f298168c040c2032cedf88d5b76affe836
+summary     : Utilities required for working with Desktop entries
+description : |
+    The Desktop File Utils package contains command line utilities for working with Desktop entries. These utilities are used by Desktop Environments and other applications to manipulate the MIME-types application databases and help adhere to the Desktop Entry Specification.
+license     :
+    - GPL-2.0-or-later
+builddeps   :
+    - pkgconfig(glib-2.0)
+setup       : |
+    %meson
+build       : |
+    %meson_build
+install     : |
+    %meson_install
```

Comment:
A trailing newline is always welcome here, most of the time you can set this in your editor of choice :slightly_smiling_face: